arch/risc-v/src/mpfs/mpfs_ihc.c

```diff
@@ -1248,11 +1248,9 @@ int mpfs_ihc_init(void)
  *
  ****************************************************************************/
 
-uintptr_t up_addrenv_va_to_pa(FAR void *va_)
+uintptr_t up_addrenv_va_to_pa(FAR void *va)
```

Comment:
```suggestion
uintptr_t up_addrenv_va_to_pa(void *va)
```

Reply:
Fixed that as well, now absolutely no FARs.